File: C/Colleen.c
#include <stdio.h>
/*
	Comment outside main
*/
void function(char *data) {printf(data,data,10,34);}
int main() {
/*
	Comment inside main
*/
function("#include <stdio.h>%2$c/*%2$c	Comment outside main%2$c*/%2$cvoid function(char *data) {printf(data,data,10,34);}%2$cint main() {%2$c/*%2$c	Comment inside main%2$c*/%2$cfunction(%3$c%1$s%3$c);}");}

File: Python/Colleen.py
# Comment outside main
display = print
if __name__ == '__main__':
    # Comment inside main
    data = """# Comment outside main
display = print
if __name__ == '__main__':
    # Comment inside main
    data = {0}{1}{0}
    display(data.format(3*chr(34), data))"""
    display(data.format(3*chr(34), data))
